airbyte-integrations/connectors/source-github/source_github/schemas/comments.json

```diff
@@ -27,18 +27,78 @@
       "type": ["null", "integer"]
     },
     "created_at": {
-      "type": ["null", "string"],
+      "type": "string",
       "format": "date-time"
     },
     "updated_at": {
-      "type": ["null", "string"],
+      "type": "string",
       "format": "date-time"
     },
     "issue_url": {
       "type": ["null", "string"]
     },
     "author_association": {
       "type": ["null", "string"]
+    },
+    "reactions": {
+      "$ref": "reactions.json"
+    },
+    "performed_via_github_app": {
+      "type": ["null", "object"],
+      "properties": {
+        "id": {
```

Comment:
99% of all fields in all our schemas for all source connectors have `["null", "type"]`, even when it's non-null in reality.

We try to be more precise about non-nullability only for:
1. primary key field(s)
2. cursor field
3. All other fields were you 100% sure about non-nullability

For other cases, we usually add `["null", "type"]` just because it's safer.
For us, it's much cheaper to have NULL in the database than be killed trying to insert null in a non-nullable column.

That's a way we are not very strict for such cases and usually automatically add `["null", "type"]`